challenge/src/components/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {

```

Comment:
this is somewhat extensible since someone could extend the button class component, but we could make things a little easier if we take in the `handleClick` function through props. that way some one could easily modify the behavior by doing something like `<Button handleClick={function(){alert('test')}} />` and reuse this code to make an alert button.